diagnostic.md

```diff
@@ -23,6 +29,7 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
+$
```

Comment:
A few possible ways are `$('li').parent().parent().find('form')`, `$('li').parent().parent().children('form')`, and `$('.listerItem').parent().parent().$('form')`.